scripts/entity/components/input_components/player_input_component.gd

```diff
@@ -2,9 +2,9 @@ class_name PlayerInputComponent extends BaseInputComponent
 ## An extension to the [BaseInputComponent] class which takes in input from a player input device.
 
 
-const JUMP_INPUT_NAME = "jump"
-const LEFT_INPUT_NAME = "left"
-const RIGHT_INPUT_NAME = "right"
+const JUMP_INPUT_NAME: StringName = "jump"
+const LEFT_INPUT_NAME: StringName  = "left"
+const RIGHT_INPUT_NAME: StringName = "right"
```

Comment:
Proper definition is &"jump" for StringNames I believe. This converts a string to a StringName

Reply:
changed